y_pef_schedule.install

```diff
@@ -12,8 +12,8 @@ use Drupal\field\Entity\FieldConfig;
  * Implements hook_install().
  */
 function y_pef_schedule_install(): void {
-  // Create custom fields required by the module.
-  _y_pef_schedule_create_fields();
+  // Update display settings for the 'activity color' field.
+  _y_pef_schedule_update_displays_on_install();
```

Comment:
this is a major change by definition, you changed install process
So module will get major version bump after merge

Reply:
we are in rc phase, so I assume we are safe, but we would need to describe after rc6 we have broken upgrade path ( possibly )